lib/ebanx/command/transfer_commit.rb

```diff
@@ -0,0 +1,16 @@
+module Ebanx
+  module Command
+    class TransferCommit < Command
+      def initialize(params)
+        @params         = params
+        @request_method = :post
+        @request_action = 'transfer/commit'
+        @response_type  = :json
+      end
+
+      def validate
+        validate_presence_or :transfer_code, :uid
```

Comment:
We have to change this logic, here and in Cancel/Retrieve.

You can use transfer_code, uid, external_referente or settlement_code